src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java

```diff
@@ -118,6 +136,7 @@ static void setupSourceProjectData(@Nonnull final SphereClient sourceProjectClie
                 ofEnglish("V-neck Tee"),
                 ofEnglish("v-neck-tee"),
                 ProductVariantDraftBuilder.of().key(RESOURCE_KEY).sku(RESOURCE_KEY).build())
+            .state(state)
```

Comment:
I would rename the `productDraft` variable to differentiate that this is not just a normal productDraft but rather it has something called state attached to it.

Reply:
This is a normal product draft :) I also don't call it `productDraftWithKey` because I attach the key to it :) Let's keep it simple.